lib/modules/manager/pip_requirements/extract.ts

```diff
@@ -121,7 +122,8 @@ export function extractPackageFile(
       const currentValue = currVal?.trim();
       dep = {
         ...dep,
-        depName,
+        depName: normalizeDepName(depName),
+        packageName: depName,
```

Comment:
shouldn't we only normalize package name? that I'd used by datasource, depName is for visualize

Reply:
I'm not sure. It's not documented in the interface.

Reply:
When `alertPackageRule` is generated, `depName` is used in branch name. I believe it should be normalized. Both `Pillow` and `pillow` and IIRC updates are grouped by branch names, would this mean that every valid name would generate a separate PR?

```
  "alertPackageRules": [
    {
      "matchDatasources": [
        "pypi"
      ],
      "matchPackageNames": [
        "Pillow"
      ],
      "matchCurrentVersion": "== 10.0.1",
      "matchFileNames": [
        "requirements.txt"
      ],
      "allowedVersions": "==10.2.0",
      "isVulnerabilityAlert": true,
      "force": {
        ...
        "branchTopic": "{{{datasource}}}-{{{depName}}}-vulnerability",
        "prCreation": "immediate",
        "enabled": true,
        "addLabels": [
          "security"
        ]
      }
    }
]
```